.github/workflows/release.yml

```diff
@@ -31,14 +26,13 @@ jobs:
           PYTHON_VERSION: ${{ matrix.python-version }}
       - name: Build wheels
         run: |
-          pip install xcffib
+          python -m pip install cffi
           python setup.py bdist_wheel
-          auditwheel repair --plat manylinux2014_x86_64 -w output_wheels dist/qtile-*.whl
```

Comment:
The error you posted seems like it is saying the platform here is too old. At the time this was made, we were able to have it generate the `manylinux2014` wheels, but this may need to be bumped to `manylinux_2_28_x86_64`. IIRC, if an older version would work, the logs would say so at that point (and I think it would generate the new requested version and the older more compatible wheels). It may also be possible to drop the `--plat` argument entirely here.